modules/drivers/vertica/test/metabase/test/data/vertica.clj

```diff
@@ -18,7 +18,10 @@
 
 (sql-jdbc.tx/add-test-extensions! :vertica)
 
-(defmethod tx/sorts-nil-first? :vertica [_] false)
+;; In ORDER BY clause, nulls come last for FLOAT, STRING, and BOOLEAN columns, and first otherwise
+;; https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/AnalyzingData/Optimizations/NULLPlacementByAnalyticFunctions.htm#2
+(defmethod tx/sorts-nil-first? :vertica [_ base-type] (not (contains? #{:type/Text :type/Boolean :type/Float}
+                                                                      base-type)))
```

Comment:
Probably worth having the body of this method on its own line

Reply:
Fixed